DB/Item.lua

```diff
@@ -165,4 +165,83 @@ function Item:HasZone(entry)
 	return false
 end
 
+function Item:IsCollectionItem(entry)
+	return entry.method == CONSTANTS.DETECTION_METHODS.COLLECTION
+end
+
+function Item:IsUsingCollectionProperties(entry)
```

Comment:
Is this type of naming an accurate description? IIRC in the first version I named those methods ``IsValidXYZ`` because the function would return whether the sum of the item's properties amount to a valid item of the given type. ``IsUsingXyzProperties`` sounds like there's some sort of provider for Xyz properties?

Not sure what to make of this, perhaps we can find a better description for these types of validation methods? I thought of them as similar to unit tests, though not explicitly labeled as such, but I figure either way we should keep names consistent and agree on a convention here :)

Reply:
Rewrote most of the validations, so they generally only use Item:IsXYZItem(entry) and Item:IsValidXYZItem(entry)